.clang-tidy

```diff
@@ -0,0 +1,65 @@
+---
+Checks:          '-*,readability-identifier-naming'
+WarningsAsErrors: ''
+HeaderFilterRegex: ''
+AnalyzeTemporaryDtors: false
+FormatStyle:     file 
+User:            zigen
+CheckOptions:
+  - key:             cert-dcl16-c.NewSuffixes
+    value:           'L;LL;LU;LLU'
+  - key:             cert-oop54-cpp.WarnOnlyIfThisHasSuspiciousField
+    value:           '0'
+  - key:             cppcoreguidelines-explicit-virtual-functions.IgnoreDestructors
+    value:           '1'
+  - key:             cppcoreguidelines-non-private-member-variables-in-classes.IgnoreClassesWithAllMemberVariablesBeingPublic
+    value:           '1'
+  - key:             google-readability-braces-around-statements.ShortStatementLines
+    value:           '1'
+  - key:             google-readability-function-size.StatementThreshold
+    value:           '800'
+  - key:             google-readability-namespace-comments.ShortNamespaceLines
+    value:           '10'
+  - key:             google-readability-namespace-comments.SpacesBeforeComments
+    value:           '2'
+  - key:             modernize-loop-convert.MaxCopySize
+    value:           '16'
+  - key:             modernize-loop-convert.MinConfidence
+    value:           reasonable
+  - key:             modernize-loop-convert.NamingStyle
+    value:           CamelCase
+  - key:             modernize-pass-by-value.IncludeStyle
+    value:           llvm
+  - key:             modernize-replace-auto-ptr.IncludeStyle
+    value:           llvm
+  - key:             modernize-use-nullptr.NullMacros
+    value:           'NULL'
+  - key:             readability-identifier-naming.AbstractClassCase
+    value:           CamelCase
+  - key:             readability-identifier-naming.ClassCase
+    value:           CamelCase
+  - key:             readability-identifier-naming.EnumCase
+    value:           CamelCase
+  - key:             readability-identifier-naming.StructCase
+#    value:           CamelCase
+    value:           aNy_CasE
+  - key:             readability-identifier-naming.TypeAliasCase
+    value:           CamelCase
+#  - key:             readability-identifier-naming.ConstantCase
+#    value:           CamelCase
+#  - key:             readability-identifier-naming.ConstantPrefix
+#    value:           'k'
+#  - key:             readability-identifier-naming.ConstantParameterCase
+#    value:           lower_case
+#  - key:             readability-identifier-naming.ConstantParameterPrefix
+#    value:           ''
+#  - key:             readability-identifier-naming.TypedefCase
+#    value:           CamelCase
+#  - key:             readability-identifier-naming.VariableCase
+#    value:           lower_case
+#  - key:             readability-identifier-naming.ParameterCase
+#    value:           lower_case
+#  - key:             readability-identifier-naming.FunctionCase
+#    value:           camelBack
```

Comment:
I set many options to enforce the naming, but I won't fix this now because I want to keep each PR small.

Reply:
OK